engines/tika/src/main/java/org/alfresco/transform/tika/parsers/ExifToolParser.java

```diff
@@ -76,13 +77,34 @@ public class ExifToolParser extends ExternalParser {
     private String separator;
 
     public ExifToolParser() {
+        this(null);
+    }
+
+    public ExifToolParser(RuntimeExec exifRuntimeExec) {
         super();
         try {
             List<ExternalParser> eParsers = ExternalParsersFactory.create(getExternalParserConfigURL());
             // if ExifTool is not installed then no parsers are returned
             if (eParsers.size() > 0) {
                 ExternalParser eParser = eParsers.get(0);
-                this.setCommand(eParser.getCommand());
+
+                String[] commandToBeExecuted;

```

Comment:
Please keep to the style of the class. We already have a mix in this project and adding: "if(" rather than: "if (" adds another. There is a second example of this on line 100.

Reply:
Think I didn't miss anything :)